src/routes/docs/products/functions/templates/+page.markdoc

```diff
@@ -60,7 +60,7 @@ A comment is made to your PR about the build, unless you enable **Silent mode**.
 ---
 * Starter
 * A simple starter function that returns "Hello, world!"
-* Node.js, Python, PHP, Dart, Node.js (TypeScript), Bun, Deno, Ruby, Kotlin, C++, .NET, Java, Swift
+* Node.js, Python, PHP, Dart, Node.js (TypeScript), Bun, Deno, Ruby, Kotlin, C++, .NET, Java, Swift, Go (Golang)
```

Comment:
Let's just leave it at Go since Golang is kind of an older term and the official language is Go.